### PR TITLE
Improve documentation on TotalScatteringReduction.

### PR DESCRIPTION
**Pull request by wdzhou**
This fixes [PD #381](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/381).

Improve documentation on method [TotalScatteringReduction](https://github.com/neutrons/mantid_total_scattering/blob/ff81e54073cd3f5419b73a459aa14987aac7b66b/total_scattering/reduction/total_scattering_reduction.py#L411)

### CODE REVIEW

File: total_scattering/reduction/total_scattering_reduction.py [hunk 1]
```diff
@@ -1417,8 +1466,14 @@ def TotalScatteringReduction(config=None):
...
             OutputWorkspace=sam_corrected)
 
-    # STEP 7:  S(Q) and F(Q), bank-by-bank
-
+    #################################################################
+    # STEP 8:  S(Q) and F(Q), bank-by-bank
+    # processing includes
+    #   1. to absolute scale
+    #   2. save to S(Q)
+    #   3. self scattering correction smd save S(Q) corrected
```
Comment (Kvieta1990) [inline total_scattering/reduction/total_scattering_reduction.py:1474]: `smd`???

File: total_scattering/reduction/total_scattering_reduction.py [hunk 2]
```diff
@@ -777,8 +810,13 @@ def TotalScatteringReduction(config=None):
         # TODO: Add when we apply Qmin, Qmax cropping
         # mask_info = generate_cropping_table(qmin, qmax)
 
+    #################################################################
     # STEP 1: Subtract Backgrounds
-
+    # van       = van - ban_bg
```
Comment (Kvieta1990) [inline total_scattering/reduction/total_scattering_reduction.py:815]: what does `ban_bg` stand for? Do you mean `van_bg`?

File: total_scattering/reduction/total_scattering_reduction.py [hunk 3]
```diff
@@ -496,6 +502,15 @@ def TotalScatteringReduction(config=None):
     # alignAndFocusArgs['CropWavelengthMax'] from characterizations file
     '''
 
+    #################################################################
+    # Format run numbers and  with facility name and/or
+    # Retrieve file names from 'config'
```
Comment (Kvieta1990) [inline total_scattering/reduction/total_scattering_reduction.py:507]: How about this?

```
# Figure out experimental runs either with run numbers
# and facility name or retrieve file name from 'config'
```

File: total_scattering/reduction/total_scattering_reduction.py [hunk 4]
```diff
@@ -1097,8 +1135,11 @@ def TotalScatteringReduction(config=None):
         OutputWorkspace=van_corrected,
         SetError='zero')
 
-    # STEP 2.1: Normalize by Vanadium
-
+    #################################################################
+    # STEP 2.1: Normalize by Vanadium and
+    #           convert to Q (momentum transfer)
+    #           For sample, container, sample raw, vanadium background
+    #################################################################
```
Comment (Kvieta1990) [inline total_scattering/reduction/total_scattering_reduction.py:1142]: We should fix the following line,

[https://github.com/neutrons/mantid_total_scattering/blob/370564bd97bcc94239a69c8b0065673164954b51/total_scattering/reduction/total_scattering_reduction.py#L1158](https://github.com/neutrons/mantid_total_scattering/blob/370564bd97bcc94239a69c8b0065673164954b51/total_scattering/reduction/total_scattering_reduction.py#L1158)

to become like this,

```
# Save (sample - back) /  van_corrected
```

**Review by Kvieta1990 — CHANGES_REQUESTED** [review]
Just some tiny issues and a bit further change request. Once those comments I put get addressed, I am pretty happy to approve the PR. Thank you!

File: total_scattering/reduction/total_scattering_reduction.py (hunk 1 above)
Comment (wdzhou) [reply total_scattering/reduction/total_scattering_reduction.py:1474]: and... column-shifted-typo

**Comment by wdzhou** [comment]
@Kvieta1990 I've made changes according to your review.  Could you take a look again and merge this PR?  Thanks.

**Review by Kvieta1990 — APPROVED** [review]
Seems good to me!

**Pull request merged**